contracts/TablelandTables.sol

```diff
@@ -102,10 +103,23 @@ contract TablelandTables is
     ) private returns (ITablelandController.Policy memory) {
         address controller = _controllers[tableId];
         if (_isContract(controller)) {
-            return
-                ITablelandController(controller).getPolicy{value: msg.value}(
-                    caller
-                );
+            try ITablelandController(controller).version() returns (
+                uint256 version
+            ) {
+                if (version == 1) {
```

Comment:
seems like this should be `2`, since the "old" interface is `V1`.

Reply:
Fair. For some reason, I was baselining in my mind to version 0, but obvious the big "V1" on the old Controller cuts against that pretty decisively.

Reply:
Or call the original controller `ITablelandControllerV0`?

Reply:
Just need them to be consistent. I think "V1" is more obviously a version than "V0", so I've updated these #s.